Twig/Extension/CKEditorExtension.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
```

Comment:
@Soullivaneuh https://github.com/sonata-project/SonataFormatterBundle/pull/113/files, if merged i can rebased this PR on it and redo `make cs` and squash